app/assets/stylesheets/custom.scss

```diff
@@ -441,9 +462,104 @@ ul.edit-listings  {
 //    =========================*/
 
 .listing-create-form {
-  background: #c5eaea;
+  background: rgba(131, 194, 177, 0.6);
 }
 
 .edit-new-listing {
   margin-top: 70px;
 }
+
+#nightly_rate {
+  margin-top: 40px;
+}
+
+select#_max_guests.form-control {
+  margin-top: 10px;
+}
+
+select#_nightly_rate.form-control {
+  margin-top: 10px;
+}
+
+#_category_id{
+  margin-top: 10px;
+}
+
+body > div.col-md-12.filter > div.row > div > input.btn.btn-default {
+  align: center;
+  margin-top: 10px;
+  margin-right: 10px;
+}
+
+
+.table-header {
+  font-size: 15px;
+  font-wieght: bold;
+}
+
+
+// /* =========================
+//
+//      cart /reservations.new
+//
+//    =========================*/
+
+h4.cart-header {
+  margin-top: 100px;
+  margin-left: 80px;
+  font-size: 34px;
+}
+
+.reservation-cart {
+  background: #fbf8ed;
+  margin-top: 60px;
+  margin-right: 80px;
+  margin-left: 80px;
+}
+
+.cart {
+  background: linear-gradient(rgba(240, 240, 230, 0.5), rgba(255, 255, 255, 0.5)), url(/assets/form-pattern.png) repeat;
+  padding-bottom: 200px;
+  margin-top: 100px;
+  margin-right: 150px;
+  margin-left: 150px;
+  margin-bottom: 40px;
+}
+
+
+// /* =========================
+//
+//      user listing show
+//
+//    =========================*/
+
+.listing-header {
+  margin-top: 100px;
+}
+
+.show-search {
+  margin-bottom: 60px;
+  margin-top: 20px;
+}
+
+.show-listing-info {
+  background: #fbf8ed;
```

Comment:
Color literals like `#fbf8ed` should only be used in variable declarations; they should be referred to via variable everywhere else.